src/main/java/com/google/javascript/gents/ModuleConversionPass.java

```diff
@@ -118,8 +133,8 @@ public void visit(NodeTraversal t, Node n, Node parent) {
                 new Node(Token.EXPORT_SPEC, Node.newString(Token.NAME, localName))));
             parent.addChildAfter(export, n);
             // Registers symbol for rewriting local uses.
-            registerLocalSymbol(child.getSourceFileName(), exportedNamespace, exportedNamespace,
-                localName);
+            registerLocalSymbol(
```

Comment:
Maybe for the future, it'd be nicer if you sent one "reformat everything" CL, instead of reformatting files as you touch them. This makes it quite hard to review the code, and will cause further confusion in blame lines (why was this line changed as part of a commit that did X?).